### PR TITLE
switching branch for the Login Ajax Session

### PR DESCRIPTION
**Pull request by hammour**
@tinydinosaurs i just removed the old pull and created a new branch.

### CODE REVIEW

**Comment by tinydinosaurs** [comment]
👎  I found a few issues:
1. need to import Navigation (which I'm currently working on, but I think it won't break if you include it). Add `<Navigation />`  right below `<section>`
2. I think `<section>` should have `className='container'`
3. "Login" text next to opening form needs to be in `<h1>` tags
4. `browserHistory` should push to /stories, not /dashboard

**Comment by hammour** [comment]
@tinydinosaurs as i don't have the guidelines for the design i have done what you asked, but note two things. 
the Navigation bar will be duplicated on the page, and also please refer to app.js (line 8 and 9). 
Second issue that we will have is the the Navigation should not live inside the section should be on a div outside the section. 
let me know if you want me to revert those two or do them differently.

**Comment by tinydinosaurs** [comment]
You're totally right about Navigation, sorry.

As far as the styles, the `seven` and `columns` classes are skeleton grid classes. The form isn't a grid component, so probably doesn't need them.

The login should definitely be nested in `<h1>` tags, because all the page titles are.

Go ahead and change those two things as well as `browserHistory.push('/stories')`

**Comment by hammour** [comment]
@tinydinosaurs added the `<h1>`, changed dashboard to Stories and the other two already done and pushed. and the `seven` and `columns` works for the section but already removed them anyway. 
let me know if you need anything else.

File: public/scripts/components/pages/Login.js [hunk 1]
```diff
@@ -1,17 +1,61 @@
...
+	$.ajax({
+		url: '/auth/login',
+		type: 'POST',
+		data: {
+			email: this.refs.email.value,
+			password: this.refs.password.value
+		},
+		// headers: {
+		// 	Accept: 'application/json'
+		// },
```
Comment (alarner) [inline public/scripts/components/pages/Login.js]: Delete the header comments then 👍

**Comment by alarner** [comment]
💯 👍 ⚡

**Pull request merged**